Sources/TuistLoader/Models+ManifestMappers/Project+ManifestMapper.swift

```diff
@@ -66,6 +69,22 @@ extension TuistCore.Project {
                 xcodeProjPath: xcodeProjPath,
                 name: name,
                 organizationName: organizationName,
+                developmentRegion: developmentRegion,
```

Comment:
the same thing about leveraging value semantics of `TuistCore.Project` from below